test/camera/camera_test.py

```diff
@@ -0,0 +1,391 @@
+        camera = Camera(camera_type=RELATIVE)
+        camera.pan(-90)
+
+        self.assertEqual(camera._current_pan_angle, 0)
+
+
+    def test_pan_right_relative_beyond_bound(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.pan(-91)
+
+        self.assertEqual(camera._current_pan_angle, 0)
+
+
+    def test_tilt_up_absolute(self):
+        camera = Camera()
+        camera.tilt(90)
+
+        self.assertEqual(camera._current_tilt_angle, 90)
+
+
+    def test_tilt_up_absolute_at_bound(self):
+        camera = Camera()
+        camera.tilt(150)
+
+        self.assertEqual(camera._current_tilt_angle, 150)
+
+
+    def test_tilt_up_absolute_beyond_bound(self):
+        camera = Camera()
+        camera.tilt(151)
+
+        self.assertEqual(camera._current_tilt_angle, 150)
+
+
+    def test_tilt_down_absolute(self):
+        camera = Camera()
+        camera.tilt(90)
+        camera.tilt(60)
+
+        self.assertEqual(camera._current_tilt_angle, 60)
+
+
+    def test_tilt_down_absolute_at_bound(self):
+        camera = Camera()
+        camera.tilt(90)
+        camera.tilt(0)
+
+        self.assertEqual(camera._current_tilt_angle, 0)
+
+
+    def test_tilt_down_absolute_beyond_bound(self):
+        camera = Camera()
+        camera.tilt(90)
+        camera.tilt(-1)
+
+        self.assertEqual(camera._current_tilt_angle, 0)
+
+
+    def test_tilt_up_relative(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.tilt(90)
+
+        self.assertEqual(camera._current_tilt_angle, 90)
+
+
+    def test_tilt_up_relative_at_bound(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.tilt(150)
+
+        self.assertEqual(camera._current_tilt_angle, 150)
+
+
+    def test_tilt_up_relative_beyond_bound(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.tilt(151)
+
+        self.assertEqual(camera._current_tilt_angle, 150)
+
+
+    def test_tilt_down_relative(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.tilt(90)
+        camera.tilt(-30)
+
+        self.assertEqual(camera._current_tilt_angle, 60)
+
+
+    def test_tilt_down_relative_at_bound(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.tilt(90)
+        camera.tilt(-90)
+
+        self.assertEqual(camera._current_tilt_angle, 0)
+
+
+    def test_tilt_down_relative_beyond_bound(self):
+        camera = Camera(camera_type=RELATIVE)
+        camera.tilt(90)
+        camera.tilt(-91)
+
+        self.assertEqual(camera._current_tilt_angle, 0)
+
+
+    def test_get_pan_angle(self):
+        camera = Camera()
+        camera.pan(90)
+
+        self.assertEqual(camera._current_pan_angle, camera.get_pan_angle())
+
+
+    def test_get_tilt_angle(self):
+        camera = Camera()
+        camera.tilt(90)
+
+        self.assertEqual(camera._current_tilt_angle, camera.get_tilt_angle())
+
+
+    def test_calibrate_pan_absolute_increase(self):
+        camera = Camera()
+
```

Comment:
Can you remove the blank space between these in all your tests? I think it looks a lot cleaner for the two line tests.

Reply:
can do.